Troubleshooting: add Fluent UI topic (#5295)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/30 3rd-Party Frameworks Issues/10 Fluent UI/05 Panels, Modals, Dialogs.md

> If you use controls like Fluent UI <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/panel" target="_blank">Panel</a>, <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/modal" target="_blank">Modal</a>, or <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog" target="_blank">Dialog</a> with DevExtreme components, you may experience issues listed below.

Reviewer: ```suggestion
If you use controls like Fluent UI <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/panel" target="_blank">Panel</a>, <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/modal" target="_blank">Modal</a>, or <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog" target="_blank">Dialog</a> with DevExtreme components, you may experience issues listed below.
```
How many issues? you may experience one of the issues listed below or several of the issues listed below?

Author: Several issues

Reviewer: Then rewrite to:
components, you may experience several of the following issues.

#### Issue 

> An overlay component (Popup, Popover, Tooltip) is behind Fluent UI Panel, Modal, or Dialog.

Reviewer: I don't understand what you mean by "behind Fluent UI Panel,.."

Author: Fluent UI Panel (or Modal, or Dialog) are overlay components made by Microsoft. Some people use them with our overlay components, and they want our components to be displayed over the Microsoft components.

Maybe "an overlay component is **displayed** behind..." would be better?

Reviewer: yes, rewrite it so

#### Solution

> In Fluent UI components, z-index is set to `1000000`. Call the [baseZIndex(zIndex)](/Documentation/ApiReference/Common/Utils/ui/dxOverlay/#baseZIndexzIndex) method to increase z-indexes of overlay components in the application:

Reviewer: ```suggestion
In Fluent UI components, the z-index is set to `1000000`. Call the [baseZIndex(zIndex)](/Documentation/ApiReference/Common/Utils/ui/dxOverlay/#baseZIndexzIndex) method to increase overlay component z-indexes in the application:
```

    <!-- tab: App.js -->
    import { baseZIndex } from 'devextreme/ui/overlay';
    baseZIndex(1000001);

#### Issue 

> DevExtreme components that use drop-down windows or overlays are not responsive when added to Fluent UI Panel, Modal, or Dialog.

Reviewer: ```suggestion
DevExtreme components that use drop-down windows or overlays are not responsive when added to the Fluent UI Panel, Modal, or Dialog.
```

#### Solution

> To resolve the issue, disable FocusTrapZone's focus trapping behavior:

Reviewer: ```suggestion
To resolve the issue, disable FocusTrapZone focus trapping behavior:
```

    <!-- tab: App.js -->
    // ...

    const focusTrapZoneProps = {
        disabled: true
    };

    function App() {
        return (
            <Panel ...
                focusTrapZoneProps={focusTrapZoneProps}
            />
        );
    }
    
    export default App;

> For Fluent UI Dialog specify the `modalProps` property, since Dialog does not include `focusTrapZoneProps`:

Reviewer: ```suggestion
For the Fluent UI Dialog, specify the `modalProps` property, since the Dialog does not include `focusTrapZoneProps`:
```

Reviewer: What do you mean by "For the Fluent UI Dialog"?

Author: I mean "if you use Fluent UI Dialog". I can change it if it's better

Reviewer: ```suggestion
To use the Fluent UI Dialog, specify the `modalProps` property, since the Dialog does not include `focusTrapZoneProps`:
```
